sigpy/mri/samp.py

```diff
@@ -199,3 +199,69 @@ def _poisson(nx, ny, K, R, calib, seed):
          int(nx / 2 - calib[-1] / 2):int(nx / 2 + calib[-1] / 2)] = 1
 
     return mask
+
+
+def spiral(fov, N, f_sampling, R, ninterleaves, alpha, gm, sm):
+    """Generate variable density spiral trajectory.
+
+    Args:
+        fov (float): field of view in meters.
+        N (int): effective matrix shape.
+        f_sampling (float): undersampling factor in freq encoding direction.
+        R (float): undersampling factor.
+        ninterleaves (int): number of spiral interleaves
+        alpha (float): variable density factor
+        gm (float): maximum gradient amplitude (T/m)
+        sm (float): maximum slew rate (T/m/s)
+
+    Returns:
+        array: spiral coordinates.
+
+    References:
+        Dong-hyun Kim, Elfar Adalsteinsson, and Daniel M. Spielman.
+        'Simple Analytic Variable Density Spiral Design.' MRM 2003.
+
+    """
+    res = fov/N
+
+    gamma = 2.678e8  # in rad/T/s
```

Comment:
Can you put `gamma` as a option and set 2.678e8 as default? This will allow users to use it for other resonant frequencies.